src/module-elasticsuite-catalog/Plugin/Ui/Category/Form/DataProviderPlugin.php

```diff
@@ -132,6 +213,25 @@ private function getAttributes($category)
             ->addStoreLabel($category->getStoreId())
             ->setOrder('position', 'ASC');
 
+        $storeId = $this->getStoreId($category);
+
+        if ($storeId) {
+            /** @var \Smile\ElasticsuiteCatalog\Model\ResourceModel\Product\Fulltext\Collection $fulltextCollection */
+            $fulltextCollection = $this->fulltextCollectionFactory->create();
+            $fulltextCollection->setStoreId($storeId)
+                ->setVisibility([Visibility::VISIBILITY_IN_CATALOG, Visibility::VISIBILITY_BOTH]);
```

Comment:
Probably not needed since it's default filter. You may want to replace it by injection of `Smile\ElasticsuiteCore\Api\Search\ContextInterface` and set properly the current Category and Store. It should then process the default filtering according to selected store and category : visibility, stock status, etc ...

Reply:
Indeed, the correct Category and Store are assigned to the ContextInterface.